### DISCUSSION TITLE
Stop trying to gamify my profile

### DISCUSSION THREAD

**Discussion by ejconlon** [community/community#18214] [Other Feature Feedback, Questions, & Ideas]
The green dot matrix for contributions was bad enough, now there's a shark with my number of merges? Just knock it off.

**Comment by 0xF6** [comment]
Dude, use this
![image](https://user-images.githubusercontent.com/13326808/172985082-686b9a49-260b-4265-85c5-3f143683e1cc.png)

And this
![image](https://user-images.githubusercontent.com/13326808/172985125-fbebb8a2-e999-43ad-acb1-bc85b8e44251.png)

**Comment by MrKrishnaAgarwal** [comment]
You can turn off Show Achievements your my profile by going into settings
